mlir/include/mlir/Dialect/SCF/Transforms/TileUsingInterface.h

```diff
@@ -97,6 +97,30 @@ struct SCFTileAndFuseOptions {
     tilingOptions = options;
     return *this;
   }
+
+  /// Control function to check if a slice needs to be fused or not,
+  /// The control function recieves
+  /// 1) the slice along which fusion is to be done,
+  /// 2) the producer value that is to be fused
+  /// 3) a boolean value set to `true` if the fusion is from
```

Comment:
Why is this necessary? Can you please provide more details on the use case you are considering?

Reply:
I dont have any right now, but trying to be future proof. Its seems useful to have the information that the fusion is happening for a destination operand. You can of course retrieve this information from the uses of `candidateSlice`